tests/Integration/Integration.Tests/PurchaseOrderModuletests/ContextBase.cs

```diff
@@ -61,6 +69,9 @@ public abstract class ContextBase
         public void EstablishContext()
         {
             this.TransactionManager = Substitute.For<ITransactionManager>();
+            this.PurchaseOrderFacadeService =
```

Comment:
Since you're mocking out the FacadeService here you could technically add some resource builder unit tests to catch the links building logic. Not a biggie though feel free to merge without